spinalcordtoolbox/resampling.py

```diff
@@ -9,37 +9,37 @@
 # About the license: see the file LICENSE.TXT
 #########################################################################################
 
+# TODO: remove resample_file (not needed)
 
 from __future__ import division, absolute_import
 
 import logging
-import nipy
 import numpy as np
-from nipy.algorithms.registration.resample import resample as n_resample
+import nibabel as nib
+from nibabel.processing import resample_from_to
+
```

Comment:
Maybe remove this extra line break?

Reply:
blank line is to separate import groups, as per [PEP8 convention](https://www.python.org/dev/peps/pep-0008/#imports)